urdf/rooster.urdf.xacro

```diff
@@ -23,7 +23,8 @@
                                              ground_camera:=false
                                              ouster_vertical_angle:=45
                                              ouster_beams:=64
-                                             ouster_hz:=10">
+                                             ouster_hz:=10
+                                             ouster_rotation=${-pi/4}">
```

Comment:
@heuristicus There was a colon missing here - this is why you had to amend the standalone to have the parameter, when it shouldn't be needed

Reply:
Good spot, thanks. I'll fix this in master rather than doing another merge.